### PR TITLE
Fixed issue #1245: changed cache backend for ZRC-DRC sync

### PR DESCRIPTION
**Pull request by stevenbal**
same issue as https://github.com/VNG-Realisatie/gemma-zaken/issues/1245 but for ZRC

### CODE REVIEW

File: src/zrc/conf/docker.py [hunk 1]
```diff
@@ -41,6 +41,11 @@ def getenv(key, default=None, required=False, split=False):
     # https://github.com/jazzband/django-axes/blob/master/docs/configuration.rst#cache-problems
     'axes_cache': {
         'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+    },
+    # Cache for BIO removal sync with DRC
+    'drc_sync': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
```
Comment (sergei-maertens) [inline src/zrc/conf/docker.py]: These files are not shared between different pods in kubernetes and we don't have a storage class that provides this currently. Can you use Redis instead?

**Comment by joeribekker** [comment]
@stevenbal Please pick this up :)

File: src/zrc/conf/docker.py [hunk 2]
```diff
@@ -41,6 +41,15 @@ def getenv(key, default=None, required=False, split=False):
     # https://github.com/jazzband/django-axes/blob/master/docs/configuration.rst#cache-problems
     'axes_cache': {
         'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+    },
+    # Cache for BIO removal sync with DRC
+    'drc_sync': {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": "redis://127.0.0.1:6379/2", # NOTE: watch out for multiple projects using the same cache!
```
Comment (sergei-maertens) [inline src/zrc/conf/docker.py]: See https://github.com/VNG-Realisatie/gemma-besluitregistratiecomponent/pull/33#pullrequestreview-273584706

Comment (sergei-maertens) [inline src/zrc/conf/docker.py]: https://github.com/VNG-Realisatie/gemma-besluitregistratiecomponent/pull/33#pullrequestreview-273584706

**Pull request merged**